pom.xml

```diff
@@ -79,7 +79,6 @@
         <apache.kafka.version>3.6.0</apache.kafka.version>
         <apache.ranger.version>2.4.0</apache.ranger.version>
         <gson.version>2.10.1</gson.version>
-        <apache.ranger.gson.version>2.10.1</apache.ranger.gson.version>
```

Comment:
There is no longer need for maintaining separate version of gson for the ranger-security extension